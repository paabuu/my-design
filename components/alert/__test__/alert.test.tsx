import 'jsdom-global/register';
import * as React from 'react';
import Alert from '../';

import { render, mount } from 'enzyme';

describe('Test Alert', () => {
    it('render correctly', () => {
        const wrapper = render(<Alert message="message" />);
        expect(wrapper).toMatchSnapshot();
    });

    it('message is bold when description is avaliable', async () => {
        const wrapper = mount(<Alert message="message" description="description" />);
        expect(wrapper).toMatchSnapshot();
    });
});
