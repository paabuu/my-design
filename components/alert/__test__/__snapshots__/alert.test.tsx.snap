// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Alert message is bold when description is avaliable 1`] = `
<Alert
  description="description"
  message="message"
  type="info"
>
  <div
    className="md-alert md-alert-info"
  >
    <p
      className="md-alert-message md-alert-message-bold"
    >
      message
    </p>
    <p
      className="md-alert-description"
    >
      description
    </p>
  </div>
</Alert>
`;

exports[`Test Alert render correctly 1`] = `
<div
  class="md-alert md-alert-info"
>
  <p
    class="md-alert-message"
  >
    message
  </p>
  <p
    class="md-alert-description"
  />
</div>
`;
